nni/runtime/log.py

```diff
@@ -7,14 +7,149 @@
 import sys
 from datetime import datetime
 from io import TextIOBase
-from logging import FileHandler, Formatter, Handler, StreamHandler
+from logging import FileHandler, Formatter, Handler, LogRecord, Logger, StreamHandler
 from pathlib import Path
+import string
 from typing import Optional
 
 import colorama
 
 from .env_vars import dispatcher_env_vars, trial_env_vars
 
+_global_logging_enabled: bool = False
+
+_root_logger: Logger = logging.getLogger()
+logging.getLogger('nni').setLevel(logging.INFO)
+
+_handlers: dict[str, Handler] = {}
+
+colorama.init()
```

Comment:
Does colorama init have side effect? What if user doesn't want colorama?

Reply:
I assumed `colorama.init()` was setting `colorama.Fore.RED`'s value. But I was wrong.
After checking the source I found `colorama.Fore.RED` is platform independent constant and `colorama.init()` overwrites `sys.stdout`.

So yes you are right. It should not be here.